test/commands/hook.spec.js

```diff
@@ -15,46 +16,62 @@ describe('hook command', () => {
   })
 
   describe('create hook', () => {
-    beforeAll(() => {
-      execa.mockReturnValue({ stdout: stubs.gitAbsoluteDir })
-      hook.create()
+    beforeEach(() => {
+      execa.mockReturnValueOnce({ stdout: stubs.relativeCoreHooksPath })
+      execa.mockReturnValueOnce({ stdout: stubs.gitAbsoluteDir })
```

Comment:
I think that here we're testing the internal implementation of `getAbsoluteHooksPath`, in this spec I would suggest to mock this module and test the implementation on the `getAbsoluteHooksPath` spec